readme/WindowsBuild.md

```diff
@@ -1,82 +1,78 @@
-# Xournal++ Windows Build
+# Windows Build
 
-![Screenshot](main-win.png?raw=true "Xournal++ Screenshot on Win10")
+![Screenshot](./main-win.png?raw=true "Xournal++ Screenshot on Win10")
+
+### Install Dependencies
+
+Xournall requires the following software to build:
 
-## Preparation
 1. Install [MSYS2](https://www.msys2.org/) to a short path without spaces.
 2. Install [NSIS](https://nsis.sourceforge.io/Download) to the standard directory.
 3. Start Mingw-w64 64bit. (Always check if it says **MINGW64** - not 32bit and not MSYS2)
 
-This will open a console. All following steps happen in this console.
+This will open a console. The following steps happen in this console:
 
-## Update MSYS2
+### Update MSYS2
 
 Do this multiple times, close the Terminal after each update
```

Comment:
This step needs to happen in the MSYS2 console and not the MINGW64 console. Could you add a clarification to the following effect?
```suggestion
Open a MSYS2 console (**not** the MINGW64 console -- close them if you have any open) and run the following command twice. Reopen the MSYS2 console each time you run the command.
```

Reply:
Sounds good :+1: 

Also any reason why the MSYS2 console has to be closed and reopened? I didn't do that and things appeared to work just fine for me - I don't use MSYS2 often

Reply:
The reason why we want to use the MSYS2 console instead of MINGW64 is that [the MINGW64 console sets environment variables (e.g., `PATH`, etc.) that may potentially break postinst scripts that get run by pacman](https://sourceforge.net/p/msys2/discussion/general/thread/dcf8f4d3/#5d86). I imagine this is not a problem for most packages, but it's better to be safe than sorry here.

> Also any reason why the MSYS2 console has to be closed and reopened

You're right, I don't think it always needs to be reopened--only if MSYS2 decides to upgrade itself and prompts the user to restart the console.